grpc-native/src/main/java/org/ballerinalang/net/grpc/builder/syntaxtree/SyntaxTreeGenerator.java

```diff
@@ -0,0 +1,351 @@
+import static org.ballerinalang.net.grpc.builder.syntaxtree.components.TypeDescriptor.getOptionalTypeDescriptorNode;
+import static org.ballerinalang.net.grpc.builder.syntaxtree.components.TypeDescriptor.getParameterizedTypeDescriptorNode;
+import static org.ballerinalang.net.grpc.builder.syntaxtree.components.TypeDescriptor.getQualifiedNameReferenceNode;
+import static org.ballerinalang.net.grpc.builder.syntaxtree.components.TypeDescriptor.getSimpleNameReferenceNode;
+import static org.ballerinalang.net.grpc.builder.syntaxtree.components.TypeDescriptor.getStreamTypeDescriptorNode;
+import static org.ballerinalang.net.grpc.builder.syntaxtree.components.TypeDescriptor.getTypeReferenceNode;
+import static org.ballerinalang.net.grpc.builder.syntaxtree.components.TypeDescriptor.getTypedBindingPatternNode;
+import static org.ballerinalang.net.grpc.builder.syntaxtree.components.TypeDescriptor.getUnionTypeDescriptorNode;
+import static org.ballerinalang.net.grpc.builder.syntaxtree.constants.SyntaxTreeConstants.SYNTAX_TREE_GRPC_ERROR_OPTIONAL;
+import static org.ballerinalang.net.grpc.builder.syntaxtree.constants.SyntaxTreeConstants.SYNTAX_TREE_VAR_STRING;
+import static org.ballerinalang.net.grpc.builder.syntaxtree.utils.CallerUtils.getCallerClass;
+import static org.ballerinalang.net.grpc.builder.syntaxtree.utils.ClientUtils.getStreamingClientClass;
+import static org.ballerinalang.net.grpc.builder.syntaxtree.utils.ClientUtils.getStreamingClientFunction;
+import static org.ballerinalang.net.grpc.builder.syntaxtree.utils.EnumUtils.getEnum;
+import static org.ballerinalang.net.grpc.builder.syntaxtree.utils.MessageUtils.getMessageNodes;
+import static org.ballerinalang.net.grpc.builder.syntaxtree.utils.ServerUtils.getServerStreamClass;
+import static org.ballerinalang.net.grpc.builder.syntaxtree.utils.ServerUtils.getServerStreamingContextFunction;
+import static org.ballerinalang.net.grpc.builder.syntaxtree.utils.ServerUtils.getServerStreamingFunction;
+import static org.ballerinalang.net.grpc.builder.syntaxtree.utils.UnaryUtils.getUnaryContextFunction;
+import static org.ballerinalang.net.grpc.builder.syntaxtree.utils.UnaryUtils.getUnaryFunction;
+import static org.ballerinalang.net.grpc.builder.syntaxtree.utils.ValueTypeUtils.getValueType;
+import static org.ballerinalang.net.grpc.builder.syntaxtree.utils.ValueTypeUtils.getValueTypeStream;
+
+/**
+ * Syntax tree generation class.
+ *
+ * @since 0.8.0
+ */
+public class SyntaxTreeGenerator {
+
+    private SyntaxTreeGenerator() {
+
+    }
+
+    public static SyntaxTree generateSyntaxTreeForServiceSample(StubFile stubFile) {
+        NodeList<ModuleMemberDeclarationNode> moduleMembers = AbstractNodeFactory.createEmptyNodeList();
+
+        NodeList<ImportDeclarationNode> imports = NodeFactory.createEmptyNodeList();
+        if (stubFile.getStubList().size() > 0) {
+            ImportDeclarationNode importForGrpc = Imports.getImportDeclarationNode("ballerina", "grpc");
```

Comment:
What is the behavior when someone uses an alias?. Something like:
```ballerina
import ballerina/grpc as balgrpc
```

Reply:
Shall we add a test for that?

Reply:
The protobuf definition file (`.proto`) does not contain any information related to ballerina. eg:
```
syntax = "proto3";

import "google/protobuf/wrappers.proto";

service helloWorld {
	rpc hello(stream google.protobuf.StringValue) returns (google.protobuf.StringValue);
}
```

Therefore this import does not change.